### DISCUSSION TITLE
importing/exporting designs or app templates

### DISCUSSION THREAD

**Discussion by yescit** [Budibase/budibase#1081] [Ideas]
It would be very cool if I could export or publish my code to a community store and importing apps from it both free or paid from within budibase builder

**Comment by yescit** [comment]
il would be a cool business model too, envato-like

**Comment by mjashanks** [comment]
Watch this space ;)

A marketplace of integrations, components and app templates is in the plan. Can't give you a date, but **not** in the first half of this year.

**Reply by yescit** [reply]
thank you very much indeed! a roadmap would be great too! Best regards
